src/rebuild.ts

```diff
@@ -53,22 +53,30 @@ class Rebuilder {
     this.lifecycle.emit('start');
 
     const rootPackageJson = await readPackageJson(this.buildPath);
-    const markWaiters: Promise<void>[] = [];
-    const depKeys = [];
+    const markWaiters: Promise<void>[] = this.types.reduce((acc: string[], curr: string) => {
```

Comment:
hi @MarshallOfSound. thanks for a quick response. 
here I tried to avoid additional variable which is not needed in the future, but can revert that ofc.